package io.github.yuqiancao19.budget;

import android.os.Bundle;

import com.getcapacitor.BridgeActivity;

public class MainActivity extends BridgeActivity {
    @Override
    public void onCreate(Bundle savedInstanceState) {
        registerPlugin(WidgetBridgePlugin.class); // must come before super.onCreate
        super.onCreate(savedInstanceState);
    }
}
